pkg/odo/cli/component/describe.go

```diff
@@ -32,7 +32,7 @@ var describeCmd = &cobra.Command{
 		} else {
 			componentName = context.Component(args[0])
 		}
-		componentDesc, err := component.GetComponentDesc(client, componentName, applicationName)
+		componentDesc, err := component.GetComponentDesc(client, componentName, applicationName, context.Project)
```

Comment:
Passing the context would have avoided changing the signature, here… ;)

Reply:
Not sure I follow :(. Passing the context where?

Reply:
If you mean pass the entire context, wouldn't that still break the signature?
I guess you mean if the context was passed to begin with :P
But then again should the context abstraction be passed down the layers?